DigitalOffice/Services/UserService/Broker/Consumers/UserConsumer.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using MassTransit;
+using LT.DigitalOffice.Broker.Requests;
+using LT.DigitalOffice.Kernel.Broker;
+using LT.DigitalOffice.UserService.Commands.Interfaces;
+using LT.DigitalOffice.Broker.Responses;
+
+namespace LT.DigitalOffice.UserService.Broker.Consumers
+{
+
+    public class UserConsumer : IConsumer<UserCredentialsInternalRequest>
+    {
+        private readonly IUserLoginCommand command;
+
+        public UserConsumer([FromServices] IUserLoginCommand command)
+        {
+            this.command = command;
+        }
+
+        public async Task Consume(ConsumeContext<UserCredentialsInternalRequest> context)
+        {
+            var response = new
+            {
+                IsSuccess = true,
+                Errors = new List<string>(),
+                Body = new { UserId = command.Execute(context.Message.Email), PasswordHash  = "fvsvsvsv"}
+            };
```

Comment:
PasswordHash? :D
That looks interesting

Reply:
I wil that fix)

Reply:
Fixed